driver.sh

```diff
@@ -149,6 +149,20 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "riscv32")
+      config=defconfig
+      image_name=vmlinux
+      using_qemu=false
+      export ARCH=riscv
+      export CROSS_COMPILE=riscv64-linux-gnu- ;;
```

Comment:
Should this be `riscv32-linux-gnu-`?

Reply:
I don't think debian even has riscv32 binutils in the repo.

Reply:
Doesn't look like it. Seems like we should only be building for riscv64 then.

Reply:
@palmer-dabbelt , any recommendations on 32b riscv binutils?